### PR TITLE
Automatically create Oracle tables

### PR DESCRIPTION
**Pull request by clark800**
This is needed for ripple-ilp-server

### CODE REVIEW

File: scripts/ci-test.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ sqlitetest() {
 oracletest() {
   # Install Oracle
   docker pull wnameless/oracle-xe-11g
-  docker run -d -p 49160:22 -p 49161:1521 wnameless/oracle-xe-11g
+  docker run -d -p 49160:22 -p 1521:1521 wnameless/oracle-xe-11g
```
Comment (MatthewPhinney) [inline scripts/ci-test.sh]: Why change the port?

Comment (clark800) [inline scripts/ci-test.sh]: The port was changed in Alan's pull request to ripple-ilp-server and we should keep it consistent.

File: src/lib/db.js [hunk 2]
```diff
@@ -17,13 +19,39 @@ function executeStatements (knex, sql) {
...
+    const url = login + (database ? '/' + database : '')
+    const env = {
+      LD_LIBRARY_PATH: '/opt/oracle/instantclient',
+      DYLD_LIBRARY_PATH: '/opt/oracle/instantclient'
+    }
+    const command = '/opt/oracle/instantclient/sqlplus'
+    const args = [url, '@' + sqlFilepath]
+    const process = spawn(command, args, {env})
+    process.on('close', (code) => {
+      return code === 0 ? resolve() : reject()
```
Comment (MatthewPhinney) [inline src/lib/db.js]: Do we log an error if the script fails?

**Comment by lumberj** [comment]
LGTM

File: src/lib/db.js (hunk 2 above)
Comment (clark800) [inline src/lib/db.js]: Not currently, we could add a message to the reject

**Comment by MatthewPhinney** [comment]
LGTM. I'll use this same logic for the notary.

**Pull request merged**